ga/20.0.0.5/kernel/helpers/build/configure.sh

```diff
@@ -5,111 +5,156 @@ fi
-then
-  if [ ! -e $keystorePath ]
+  if [ "$SSL" != "false" ] && [ "$TLS" != "false" ]
   then
-    # Generate the keystore.xml
-    export KEYSTOREPWD=$(openssl rand -base64 32)
-    sed "s|REPLACE|$KEYSTOREPWD|g" $SNIPPETS_SOURCE/keystore.xml > $SNIPPETS_TARGET_DEFAULTS/keystore.xml
-    chmod g+w $SNIPPETS_TARGET_DEFAULTS/keystore.xml
+    if [ ! -e $keystorePath ]
+    then
+      # Generate the keystore.xml
+      export KEYSTOREPWD=$(openssl rand -base64 32)
+      sed "s|REPLACE|$KEYSTOREPWD|g" $SNIPPETS_SOURCE/keystore.xml > $SNIPPETS_TARGET_DEFAULTS/keystore.xml
+      chmod g+w $SNIPPETS_TARGET_DEFAULTS/keystore.xml
+    fi
   fi
-fi
 
-# Install needed features
-if [ "$FEATURE_REPO_URL" ]; then
-  curl -k --fail $FEATURE_REPO_URL > /tmp/repo.zip
-  installUtility install --acceptLicense defaultServer --from=/tmp/repo.zip || if [ $? -ne 22 ]; then exit $?; fi
-  rm -rf /tmp/repo.zip
-else
-  installUtility install --acceptLicense defaultServer || if [ $? -ne 22 ]; then exit $?; fi
-fi
+  # Install needed features
+  if [ "$FEATURE_REPO_URL" ]; then
+    curl -k --fail $FEATURE_REPO_URL > /tmp/repo.zip
+    installUtility install --acceptLicense defaultServer --from=/tmp/repo.zip || if [ $? -ne 22 ]; then exit $?; fi
+    rm -rf /tmp/repo.zip
+  else
+    installUtility install --acceptLicense defaultServer || if [ $? -ne 22 ]; then exit $?; fi
+  fi
 
-# Apply interim fixes found in /opt/ibm/fixes
-# Fixes recommended by IBM, such as to resolve security vulnerabilities, are also included in /opt/ibm/fixes
-# Note: This step should be done once needed features are enabled and installed using installUtility.
-find /opt/ibm/fixes -type f -name "*.jar"  -print0 | sort -z | xargs -0 -n 1 -r -I {} java -jar {} --installLocation $WLP_INSTALL_DIR
-#Make sure that group write permissions are set correctly after installing new features
-find /opt/ibm/wlp -perm -g=w -print0 | xargs -0 -r chmod -R g+rw
-# Create a new SCC layer
-if [ "$OPENJ9_SCC" == "true" ]
-then
-  populate_scc.sh
-fi
-#Make folder executable for a group
-find /opt/ibm/wlp -type d -perm -g=x -print0 | xargs -0 -r chmod -R g+rwx
+  # Apply interim fixes found in /opt/ibm/fixes
+  # Fixes recommended by IBM, such as to resolve security vulnerabilities, are also included in /opt/ibm/fixes
+  # Note: This step should be done once needed features are enabled and installed using installUtility.
+  find /opt/ibm/fixes -type f -name "*.jar"  -print0 | sort -z | xargs -0 -n 1 -r -I {} java -jar {} --installLocation $WLP_INSTALL_DIR
+  #Make sure that group write permissions are set correctly after installing new features
+  find /opt/ibm/wlp -perm -g=w -print0 | xargs -0 -r chmod -R g+rw
+  # Create a new SCC layer
+  if [ "$OPENJ9_SCC" == "true" ]
+  then
+    populate_scc.sh
+  fi
+  #Make folder executable for a group
+  find /opt/ibm/wlp -type d -perm -g=x -print0 | xargs -0 -r chmod -R g+rwx
+
+  if [[ -n "$SEC_SSO_PROVIDERS" ]]; then
```

Comment:
This should be added prior to installing necessary features (line with # Install needed features), so the features being enabled by these snippets can be installed by installUtility.